docs/src/Guides/98 Migration from 4.X.md

```diff
@@ -31,13 +31,25 @@ Autocomplete *is* different. v5 encourages you to tie autocompletes to specific
 
 Autodeferring is also pretty similar, although there's more control, with options to allow for global autodefers and extension-wide ones.
 
+## Events
+
+Similarly to Slash Commands, events have also been reworked in v5. Instead of `@bot.event` and `@extension_listener`, the way to listen to events is now `@listen`. There are multiple ways to subscribe to events, whether it is using the function name or the argument of the `@listen` decorator. You can find more information on handling events using v5 [on its own guide page](../10 Events).
+
 ## Other Types of Interactions (Context Menus, Components, Modals)
 
 These should be a lot more familiar to you - many interactions in v5 that aren't slash commands are similar to v4, minus name changes (largely to the decorators and classes you use). They should still *function* similarly though, but it's never a bad idea to consult the various guides that are on the sidebar to gain a better picture of how they work.
 
 [If you're using context menus](../04 Context Menus) (IE `@bot.user_command` or `@bot.message_command`), note that the decorator you use is slightly different from either of the two older ones. You now use `@context_menu`, and specify the type of context menu through `context_type` - otherwise, it's mostly the same.
```

Comment:
We added equivalents `message_context_menu` and `user_context_menu` decorators. Can you add info there?

Reply:
Done, maybe I should add in the Context Menu guide the mention of these two decorators? (I just learned they exist lol)

Reply:
Not in this pr I think, but better ask Polls about it